src/providers/utils/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const DOC_HELP_LINK_HREF =
```

Comment:
i think we have a central doc links location
@avivtur do we?

Reply:
I didn't find one so moved it to `src/utils/constants` for now.

Reply:
u have in src/utils/links.ts

Reply:
It wasn't there when I worked on it...